src/app/components/tables/data-table/terra-data-table.component.html

```diff
@@ -24,18 +24,23 @@
                 <th *ngIf="!header.isHidden"
                     width="{{header.width}}px"
                     [ngStyle]="{ 'text-align': getTextAlign(header) }"
+                    [class.sortable]="inputIsSortable && header?.sortBy"
```

Comment:
Warum kein ngClass?

Reply:
ist so schöner/kürzer und ist ngClass nur in einem anderen Syntax
Als vergleich `[ngClass]="{ 'sortable': inputIsSortable && header?.sortBy }"`